eth/ethconfig/config.go

```diff
@@ -55,21 +55,22 @@ var DefaultConfig = NewDefaultConfig()
 
 func NewDefaultConfig() Config {
 	return Config{
-		NetworkId:                 0, // enable auto configuration of networkID == chainID
-		StateHistory:              params.FullImmutabilityThreshold,
-		TrieCleanCache:            512,
-		TrieDirtyCache:            256,
-		TrieDirtyCommitTarget:     20,
-		TriePrefetcherParallelism: 16,
-		SnapshotCache:             256,
-		AcceptedCacheSize:         32,
-		Miner:                     miner.Config{},
-		TxPool:                    legacypool.DefaultConfig,
-		BlobPool:                  blobpool.DefaultConfig,
-		RPCGasCap:                 25000000,
-		RPCEVMTimeout:             5 * time.Second,
-		GPO:                       DefaultFullGPOConfig,
-		RPCTxFeeCap:               1, // 1 AVAX
+		NetworkId:                  0, // enable auto configuration of networkID == chainID
+		StateHistory:               params.FullImmutabilityThreshold,
+		TrieCleanCache:             512,
+		TrieDirtyCache:             256,
+		TrieDirtyCommitTarget:      20,
+		TriePrefetcherParallelism:  16,
+		SnapshotCache:              256,
+		AcceptedCacheSize:          32,
+		Miner:                      miner.Config{},
+		TxPool:                     legacypool.DefaultConfig,
+		BlobPool:                   blobpool.DefaultConfig,
+		RPCGasCap:                  25000000,
+		RPCEVMTimeout:              5 * time.Second,
+		GPO:                        DefaultFullGPOConfig,
+		RPCTxFeeCap:                1, // 1 AVAX
+		HistoricalStateQueryWindow: 43200,
```

Comment:
Since pruning is left to false, this should be for an archive node so I left the default to 43200... Should we not set this? 🤔 It seems like it's later overridden by the evm plugin config defaults anyway.